### ISSUE TITLE
How to use the request builder when the path differs?

### ISSUE THREAD

**Issue by StarpTech** [commercetools/nodejs#1338]
Hello,
how can I retrieve the active cart with the request builder? Thank you.

```js
   cart = await this.client.execute({
        uri: this.requestBuilder.myCarts...,
        method: 'GET'
    });
```

**Comment by daern91** [comment]
Hi @StarpTech,

I had a look and it does indeed seem like we are missing this functionality on the request builder. Thanks a lot for reporting it!

cc @skyriakou, we should probably put this one on the roadmap.

**Comment by daern91** [comment]
Hi again @StarpTech, please have a look at the referenced PR above. When it is merged you should be able to retrieve the active cart like below.

```js
cart = await this.client.execute({
    uri: this.requestBuilder.myActiveCart.build(),
    method: 'GET'
});
```

**Comment by daern91** [comment]
Released as `@commercetools/api-request-builder@5.3.0`
